packages/moonstone/Scrollable/Scrollable.js

```diff
@@ -485,6 +537,56 @@ class ScrollableBase extends Component {
 		this.uiRef.bounds.scrollHeight = this.uiRef.getScrollBounds().scrollHeight;
 	}
 
+	clearOverscrollEffect = (orientation, position) => {
+		const {type} = this.uiRef.getOverscrollStatus(orientation);
+
+		if (type !== overscrollTypes.none) {
+			this.overscrollJobs[orientation][position].startAfter(
+				(type === overscrollTypes.scrolling) ? overscrollTimeout : 0,
+				orientation,
+				position,
+				overscrollTypes.none,
+				0
+			);
+		}
+	}
+
+	clearAllOverscrollEffects = () => {
+		this.clearOverscrollEffect('horizontal', 'before');
+		this.clearOverscrollEffect('horizontal', 'after');
+		this.clearOverscrollEffect('vertical', 'before');
+		this.clearOverscrollEffect('vertical', 'after');
+	}
+
+	applyOverscrollEffect = (orientation, position, type, ratio) => {
+		const
+			nodeRef = this.overscrollRefs[orientation],
+			prefix = '--scrollable-overscroll-ratio-';
```

Comment:
How about defining `prefix` as global in this file?

Reply:
For easy testing, I needed to define here. I will move it.